docker/build_scripts/build_utils.sh

```diff
@@ -155,7 +159,7 @@ function build_git {
     local git_sha256=$2
     check_var ${git_sha256}
     check_var ${GIT_DOWNLOAD_URL}
-    curl -sSLO ${GIT_DOWNLOAD_URL}/v${git_fname}.tar.gz
+    fetch_source v${git_fname}.tar.gz ${GIT_DOWNLOAD_URL}
```

Comment:
look like `curl` need to be specified

Reply:
There are 2 functions, this calls `fetch_source`, which just calls `_fetch_source curl "$@"`. The idea is that fetching with wget is not something that should be done unless explicitly requested, the default is curl.

Reply:
I just realized, there is `_fetch_source` and `fetch_source`

Reply:
Gotcha. If CI is green that works for me.